tsconfig.json

```diff
@@ -1,3 +1,7 @@
 {
-  "extends": "astro/tsconfigs/base"
+  "extends": "astro/tsconfigs/base",
+  "compilerOptions": {
+    // "strictNullChecks": true,
```

Comment:
should this be commented out? if so when should it be uncommented?

Reply:
I've put it in this branch as it was recommended in the astro docs for the features I've added in, but it gives errors for dataset and image pages of the website that weren't there so I've commented out. Left it there in case we want to make things stricter in the future but can delete it.